cmd/cstor-pool-mgmt/volumereplica/volumereplica.go

```diff
@@ -718,7 +718,7 @@ func GetAndUpdateSnapshotInfo(
 	//   as is to cover this corner case below check is required.
 	if cvr.Status.Phase == apis.CVRStatusOnline &&
 		len(cvr.Status.PendingSnapshots) != 0 {
-		klog.Infof("CVR: %s is marked as %s hence removing pending snapshots %v",
+		klog.V(2).Infof("CVR: %s is marked as %s hence removing pending snapshots %v",
```

Comment:
no need to change loglevel as these are once in a while logs

Reply:
Done